### ISSUE TITLE
Bring back ?use=v2

### ISSUE THREAD

**Issue by Rachel-Eichenberger** [Uniswap/interface#3120]
**Is your feature request related to a problem? Please describe.**

Users with a bad V3 pool with V2 tokens are unable to sell on current UI.
multiple pools with "Better Price" but unusable due to token (usually FOT broken V3 pool)

**Describe the solution you'd like**
Would love to have the force version "?use=V2" returned so users don't have to use the old IPFS version links

**Comment by justindomingue** [comment]
Should require no backend changes. Interface can set `Protocols` based on `use=` value

**Comment by marsrobertson** [comment]
From: https://www.reddit.com/r/UniSwap/comments/qnil9e/force_v2_swap/

### https://app.uniswap.org/#/swap?use=v2 

**Is it working though?**

I tried this morning, ended up manually approving and writing directly to the contract on Etherscan.

<img width="833" alt="Screenshot 2022-01-14 at 08 58 50" src="https://user-images.githubusercontent.com/44369284/149528506-e3b2ec70-1841-4267-b189-bd88240634fd.png">

**Comment by justindomingue** [comment]
@marsrobertson I believe the functionality was removed from the router because it automatically considers both v3 and v2 and theoretically finds the best route, removing the need for a switch.

we are talking about adding a selector in the settings / supporting the url param again though.

there might also be some work that can be done in the router itself to better support fot tokens

**Comment by Rachel-Eichenberger** [comment]
@willpote  sorry I meant to add this last night

example tokens include but not limited
New Fees.WTF Token  0xA68Dd8cB83097765263AdAD881Af6eeD479c4a33
Saitama 0x8B3192f5eEBD8579568A2Ed41E6FEB402f93f73F

**Comment by Rachel-Eichenberger** [comment]
@justindomingue 
yeah it was removed and for the most part it works, but if the pool in V3 is broke (or setup single sided with a token that will break) Then the UI thinks V3 has a better price, but the pools are un usable. So only in the rarer instance where a tokens V3 isn't set up correctly or not working.

**Comment by willhennessy** [comment]
Hi Rachel, we're working on a change in the Auto Router to gracefully fallback to V2 for fee-on-transfer tokens. https://github.com/Uniswap/swap-router-contracts/pull/44/files

**Comment by zadamg** [comment]
> Hi Rachel, we're working on a change in the Auto Router to gracefully fallback to V2 for fee-on-transfer tokens. https://github.com/Uniswap/swap-router-contracts/pull/44/files

How will this address rebasing tokens that require V2?  (Ampleforth, in particular)

**Comment by willpote** [comment]
The auto router now automatically uses V2 for fee on transfer tokens.

@zadamg unfortunately we don't have a solution for rebasing tokens currently. I've opened another issue in the auto router repo to track this.
